e2e/tests/feature-page.spec.ts

```diff
@@ -37,6 +37,26 @@ test('matches the screenshot', async ({page}) => {
   await expect(pageContainer).toHaveScreenshot();
 });
 
+// TODO: Remove this test once the usage chart is enabled by default.
+test('matches the screenshot with usage chart', async ({page}) => {
+  await page.goto('http://localhost:5555/features/odit64?showUsageChart');
```

Comment:
Nit:  use hyphens to separate the words: show-usage-chart [1]

[1] https://developers.google.com/search/docs/crawling-indexing/url-structure